civis/tests/test_parallel.py

```diff
@@ -312,6 +311,10 @@ def _get_container(job_id):
                        'max_submit_retries': mock.ANY,
                        'max_job_retries': mock.ANY,
                        'hidden': True}
+    for key in ['docker_image_name', 'docker_image_tag', 'repo_http_uri',
+                'repo_ref', 'remote_host_credential_id', 'git_credential_id',
+                'cancel_timeout', 'time_zone']:
+        expected_kwargs[key] = mock_script[key]
```

Comment:
Let's centralize this list in one spot so we don't have to maintain it in the tests and the code?